System.IO.Abstractions.sln

```diff
@@ -30,16 +28,18 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "images", "images", "{F78189
 		images\icon_256x256.png = images\icon_256x256.png
 	EndProjectSection
 EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "src", "src", "{C078E0B6-9747-475F-A999-B9E775DF6643}"
```

Comment:
The "Solution Explorer" is now quite inconsistent (only one project in directory `src`):
![image](https://user-images.githubusercontent.com/3438234/204300822-a03127bd-2885-4be6-a88b-b951e6d56df5.png)

I would suggest keep the functional projects on the outermost level and move the meta projects to a dedicated directory, e.g.:
![image](https://user-images.githubusercontent.com/3438234/204301186-b43fc915-c027-41a8-865f-453fce04e731.png)

Reply:
Oh, that's indeed messy. I cleaned it up a bit but I'm only on VS Code, please have a look :)

Reply:
Looks much better now:
![image](https://user-images.githubusercontent.com/3438234/204313573-9333f4eb-b375-4cc1-b992-d6eeb5416489.png)

Reply:
What is the Benchmarks project? Should this also have the "TestableIO" prefix?